bench/Cargo.toml

```diff
@@ -13,7 +13,8 @@ ndarray = "0.15"
 ordered-float = "3.4.0"
 criterion = "0.3"
 rand_chacha = "0.3.1"
+priority-queue = "1.3.1"
```

Comment:
Can we use [`binary_heap`](https://doc.rust-lang.org/std/collections/binary_heap/index.html) instead? The priority-queue crate is licensed under LGPL, which is not compatible with the public projects of ours, which adopt Apache-2.0.